avro2tf/src/main/scala/com/linkedin/avro2tf/helpers/HashingTransformer.scala

```diff
@@ -81,14 +81,14 @@ object HashingTransformer {
               group => {
                 val idValue = group._2.reduce {
                   (a, b) => {
-                    hashInfo.combinerType match {
+                    hashInfo.combiner match {
                       case CombinerType.SUM | CombinerType.AVG => a.copy(value = a.value + b.value)
```

Comment:
Do we also want to update the enum name of `CombinerType` to `Combiner`?

Reply:
I think so. There's a similar inconsistency with `dtype` vs `datatype`, but that inconsistency exists in Tensorflow as well.